cinch/roles/jenkins_slave_container_wrapup/templates/jswarm.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -xe
-. /etc/sysconfig/jenkins_swarm.templated
+. /var/lib/jenkins/jenkins_swarm.templated
```

Comment:
Yeah, let's use **/etc** for the config file and make necessary permissions changes.  **/opt/jswarm** is good for the jar file, and I would put the startup script into **/usr/local/bin**.